includes/integration.php

```diff
@@ -275,7 +275,6 @@ public function ensure_hooks(){
     
         // background events tracking
         add_action('woocommerce_add_to_cart', array($this, 'add_to_cart'), 10, 6);
-        add_action('woocommerce_before_cart_item_quantity_zero', array($this, 'remove_from_cart'), 10);
```

Comment:
Why are you removing this hook entirely and not replacing it with the new one `woocommerce_cart_item_removed`?

Reply:
Because the next row is adding 'woocommerce_remove_cart_item' hook witch is used to detect if item is removed from cart.